README.md

```diff
@@ -128,12 +128,26 @@ file at `~/.infrastructure_ansible_vault_pass.txt`. This is the same password as
 
 #### Production devops development
 
+> This method defaults to creating a 4Gb VirtualBox VM, which can strain an 8Gb Mac. We suggest tweaking the Vagrantfile to restrict ram usage to 2Gb at first, or using a machine with at least 12Gb ram.
+
 Install [Vagrant](http://www.vagrantup.com/), [VirtualBox](https://www.virtualbox.org) and [Ansible](http://www.ansible.com/).
 
-Install the hostsupdater plugin: `vagrant plugin install vagrant-hostsupdater`
+Install a couple of useful Vagrant plugins: `vagrant plugin install vagrant-hostsupdater vagrant-disksize`
```

Comment:
If the plugin vagrant-disksize isn't an absolute requirement but a nice to have then explain what it does and why it's useful

Reply:
Ah I see it is a new requirement to make the disk bigger automatically. Maybe get rid of the word "useful" in the sentence "Install a couple of useful Vagrant plugins". It makes it sound like they're not strictly necessary but I think that's not the case. They're kind of both essential, right?